File: Python/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

# Prompt for height until it fits parameters
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

copy = height - 1

# iterate over and overcreating pyramid

for i in range(height):
    for copy in range(copy):
        print(" ", end="")
    for k in range(i + 1):
        print("#", end="")
    print("  ", end="")
    for k in range(i + 1):
        print("#", end="")
    print()

File: C/migration/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get the height of the Pyramid (minimum 1, maximum 8)
    
    int height;
   
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
   
   
    // Create Pyramid
     
    for (int i = 0; i < height; i++)
    {
        // Determine pyramid's indentation
        
        for (int z = height - 1; z > i && height > 1; z--)
        {
            printf(" ");
        }
        
        // Determine pyramid's length
        
        for (int j = -1; j < i; j++)
        {
            printf("#");
        }
        printf("  ");
        
        for (int y = -1; y < i; y++)
        {
            printf("#");
        }
 
        printf("\n");
    }
    
}